packages/database/src/migrations/20210118014114-CreateIndividualEntityType-modifies-data.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+
+import { TupaiaDatabase } from '../TupaiaDatabase';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+exports.up = async function () {
+  const db = new TupaiaDatabase();
+  await db.executeSql(`
+    ALTER TYPE public.entity_type ADD VALUE 'individual';
```

Comment:
I wonder if adding an enum counts as modify-data or modify-schema?

Reply:
That's a very good point Nick. I think that should be a shcema.